### ISSUE TITLE
aria-hidden-focus fix does not suggest unhiding the parent element

### ISSUE THREAD

**Issue by andrewhick** [dequelabs/axe-core#4552]
### Product

axe Extension

### Product Version

4.9.1

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation

The Axe issue "ARIA hidden element must not be focusable or contain focusable elements" should account for the possibility that the ARIA hidden element could have been hidden incorrectly.

In some cases, we encounter content which is visually present but incorrectly hidden using ARIA - meaning that some users will perceive different content.

We would suggest amending the potential fix to:

To solve this problem, you need to fix one of the following:

* Focusable content should be disabled or removed from the DOM
* [ADD:] The hidden element should be made visible to assistive technologies

#### Actual

The fix only suggests the option of removing focusable content: "To solve this problem, you need to fix the following: Focusable content should be disabled or be removed from the DOM."

#### How to Reproduce

Go to ["Hidden element containing focusable elements" on this test page](https://andrewhick.com/accessibility/failures.html#hidden-focusable) and run Axe DevTools. This example reflects one which was recently found in a real-life scenario.

**Comment by straker** [comment]
Thanks for the suggestion. We'll add this to the 4.11 milestone.
